tailwind.config.js

```diff
@@ -3,7 +3,5 @@ module.exports = {
   theme: {
     extend: {},
   },
-  plugins: [
-    require('@tailwindcss/typography'),
-  ],
```

Comment:
Typography might still be useful for other pages right? Or do all pages use the same theme as Docusaurus (is that a tailwind theme?).

Reply:
It's quite easy to add it back whenever we need it. And if we create pages using docusaurus, default's docusaurus theme will apply to these pages as well (ex; the blog that used to be there before I comment it for future use).

It's documented here:  
https://tailwindcss.com/docs/typography-plugin

The instructions are quite simple.

Reply:
I didn't need typography so not adding it if unused. We can consider it again if we needed later. Resolving conversation.